doozer/doozerlib/distgit.py

```diff
@@ -1286,6 +1289,48 @@ def update_build_db(self, success_flag, task_id=None, scratch=False):
                         traceback.print_exc()
                         self.logger.error(f'Unable to extract brew task information for {task_id}')
 
+    def update_konflux_db(self, build_info, success, scratch=False):
+        if scratch:
+            return
+
+        if not self.runtime.konflux_db:
+            self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
+            return
+
+        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
+        try:
+            source_repo, commitish = build_info['source'].split('#')
+
+            build_record = KonfluxBuildRecord(
+                name=self.metadata.name,
+                group=self.runtime.group,
+                version=build_info['version'],
+                release=build_info['release'],
+                assembly=self.runtime.assembly,
+                el_target=f'el{self.metadata.branch_el_target()}',
+                arches=self.metadata.get_arches(),
+                rpms=[],  # TODO
+                parent_images=build_info['extra']['image']['parent_images'],
+                source_repo=source_repo,
+                commitish=commitish,
+                embargoed=build_info['release'].split('.')[1] == 'p1',  # TODO
+                start_time=datetime.strptime(build_info['start_time'], '%Y-%m-%d %H:%M:%S.%f'),
+                end_time=datetime.strptime(build_info['completion_time'], '%Y-%m-%d %H:%M:%S.%f'),
+                artifact_type=ArtifactType.IMAGE,
+                engine=Engine.BREW,
+                image_tag=build_info['extra']['image']['index']['pull'][0],
+                outcome=success,
+                job_url=os.getenv('BUILD_URL', 'n/a'),
```

Comment:
maybe we store brew task url here? if we're planning to store the pipeline url for konflux, then it would be similar.

Reply:
Done in 04f6e892b967742a3e7ca833ec6f017c1391fdb1